configs/anvil/config.20170926.FCT2.A_WCYCL1850S.ne30_oECv3.anvil

```diff
@@ -107,31 +124,6 @@ endYear = 22
 startYear = 1
 endYear = 9999
 
-[oceanObservations]
-## options related to ocean observations with which the results will be compared
-
-# directory where ocean observations are stored
-baseDirectory = /lcrc/group/acme/diagnostics/observations/Ocean
-
-[seaIceObservations]
-## options related to sea ice observations with which the results will be
-## compared
-
-# directory where sea ice observations are stored
-baseDirectory = /lcrc/group/acme/diagnostics/observations/SeaIce
-
```

Comment:
also this piece below about `timeSeriesSeaIceAreaVol`. And similarly for other config example files.

Reply:
Actually, it seems this may be the last config file with this issue.

Reply:
These are all default options being copied into example config files.  I realize they might demonstrate to users some of the capabilities that they could customize but we have several hundred such options at this point and these few are just arbitrary in my view.  As I said, they make changing defaults very difficult and I think they confuse and overwhelm new users rather than providing helpful direction.  That's why I think we want to keep the examples as more the minimum set of options needed to analyze that particular example run and rely on documentation, config.example and config.default to point out additional options.